presto-main/src/test/java/io/prestosql/sql/planner/iterative/rule/TestPruneAggregationColumns.java

```diff
@@ -74,7 +74,7 @@ private ProjectNode buildProjectedAggregation(PlanBuilder planBuilder, Predicate
                 planBuilder.aggregation(aggregationBuilder -> aggregationBuilder
                         .source(planBuilder.values(key))
                         .singleGroupingSet(key)
-                        .addAggregation(a, planBuilder.expression("count()"), ImmutableList.of())
-                        .addAggregation(b, planBuilder.expression("count()"), ImmutableList.of())));
```

Comment:
Would https://errorprone.info/bugpattern/StaticQualifiedUsingExpression catch this?

Reply:
Yep

Reply:
I added this check to set.